lib/model/om/BaseObject.php

```diff
@@ -185,6 +185,11 @@ public function __isset($name)
       }
     }
 
+    if ('aclPermissions' == $name)
```

Comment:
(Same as previous comment.)

Reply:
Fixed.

Reply:
Based on our discussion below, I have reverted my fix to re-order these items.